static/src/javascripts-legacy/bootstraps/enhanced/media/video-player.js

```diff
@@ -10,9 +10,7 @@ define([
     // In rjs this module is shimmed in the config
     // https://github.com/guardian/frontend/blob/webpack-redux/tools/__tasks__/compile/javascript/rjs.config.js#L42
     // We can move this up to the define() when we go 100% WP
```

Comment:
Can we action this comment now, or will there be a forthcoming PR to deal with comments like this?